pyvista/utilities/geometric_objects.py

```diff
@@ -161,14 +161,15 @@ def CylinderStructured(radius=0.5, height=1.0,
     grid.dimensions = [nr, theta_resolution+1, z_resolution]
 
     # Orient properly in user direction
-    direction = np.array(direction)
-    direction = direction / np.linalg.norm(direction)
-    vx = np.random.randn(3)
-    vx -= vx.dot(direction) * direction
-    vx /= np.linalg.norm(vx)
-    vy = np.cross(direction, vx)
-    rmtx = np.array([vx, vy, direction])
-    grid.points = grid.points.dot(rmtx)
+    vx = np.array([0., 0., 1.])
```

Comment:
This is the direction in which I created the structured grid to begin with, so my logical here is that if the direction is anything but this, then let's rotate from this vector to the given direction